### ISSUE TITLE
Support additional token for fix commits

### ISSUE THREAD

**Issue by markphip** [wearerequired/lint-action#92]
When using this action (in my case with Python black) if it makes and pushes a fix commit this does not trigger a new build .. freezing my workflow. This is because events triggered using the github action token do not trigger new events. It ought to be possible to provide an additional, optional, github token to use for the commit/push of a fix.

I tried passing this token in place of github action token but that only causes your calls to checks API to fail with a 403.

**Comment by markphip** [comment]
Discovered there is an easy solution here .. so no changes are needed.

```
    steps:
    - uses: actions/checkout@v2
      with:
        token: ${{ secrets.BOT_TOKEN }} # this is so commit of any fixes are done by this user
```
Closing issue
